docs/migrate.md

```diff
@@ -4,6 +4,19 @@ This document records manual migrations that are necessary to apply when upgradi
 Sourcegraph versions. All manual migrations between the version you are upgrading from and the
 version you are upgrading to should be applied (unless otherwise noted).
 
+## 3.15
+
+### Existing installations: Migrating lsif-server
+
+Version 3.15 splits lsif-server into multiple services, but retains the same persistent volume. In order for the
+update to succeed, the old lsif-server service and deployment must be detached from the persistent volume. The
+following commands need to be run once after the updates have been applied to your cluster:
+
+```shell script
+kubectl delete svc lsif-server
+kubectl delete deployment lsif-server
```

Comment:
Maybe I am wrong, but I thought `kubectl-apply-all.sh` would automatically do this?

Reply:
It doesn't seem to!

cc @uwedeportivo 

See https://github.com/kubernetes/kubernetes/issues/66430 and also https://github.com/sourcegraph/deploy-sourcegraph/pull/570 which uses -k instead of -f, which doesn't respect prune (apparently).

Reply:
When running this on a test cluster the unreferenced services never went away (unless I missed a reference somewhere...?).

Reply:
Ah, got it, I must be wrong then. Seems reasonable to me, then.

Reply:
I'm not sure if this is affecting Pulumi in CI since we can't/aren't doing it in the automated tests.

Reply:
https://buildkite.com/sourcegraph/deploy-sourcegraph/builds/2655#4134e558-9765-4e96-9a2c-4a5104fdbd9e

Reply:
Oof ok nevermind. The reason that CI is failing is noted in a comment below.